ios/FluentUI/PreviewCard/PreviewCard.swift

```diff
@@ -28,38 +28,64 @@ import SwiftUI
     private var previewCardView: PreviewCard!
 }
 
+extension View {
+    /// Sets the height and width of the Preview Card.
+    func cardBorderModifiers() -> some View {
+        self
+            .frame(minWidth: PreviewCard.Constants.cardMinWidth, minHeight: PreviewCard.Constants.cardMinHeight)
+    }
+    /// Sets the background border style, radius, color, and linewidth of the Preview Card.
+    func cardBackgroundModifiers() -> some View {
+        self
+            .background(
+                RoundedRectangle(cornerRadius: PreviewCard.Constants.cardCornerRadius)
+                    .strokeBorder(Color(Color.RGBColorSpace.sRGB, red: PreviewCard.Constants.cardStrokeBorderColor, green: PreviewCard.Constants.cardStrokeBorderColor, blue: PreviewCard.Constants.cardStrokeBorderColor), lineWidth: PreviewCard.Constants.cardLineWidth)
+    )}
+}
+
 /// View that represents the PreviewCard.
 public struct PreviewCard: View {
-
+    var showEleavatedCard: Bool
     @Environment(\.theme) var theme: FluentUIStyle
     @Environment(\.windowProvider) var windowProvider: FluentUIWindowProvider?
     /// Creates inner PreviewCard view.
     @ViewBuilder
     var innerContents: some View {
         HStack {
             Spacer()
-                .frame(minWidth: Constants.cardMinWidth, minHeight: Constants.cardMinHeight)
+                .cardBorderModifiers()
+                .cardBackgroundModifiers()
+                .modifyIf(showEleavatedCard, { view in view
+                        .shadow(color: Color(.black.withAlphaComponent(0.14)),
```

Comment:
You can put these alpha component values as constants so that they're easier to reference/change later if needed